spdata/reader.py

```diff
@@ -0,0 +1,38 @@
+"""Various methods for reading data"""
+
+import numpy as np
+
+import spdata.types as ty
+
+
+def _parse_metadata(line: str) -> (int, int):
```

Comment:
We don't need label parsing for now (and preemptively Z coordinate)?

Reply:
That's why I asked you for review - I was not fully sure with this.

Reply:
IMO if this we have a port of some other code in C#, then 100% of its functionalities should be provided in final code.

Reply:
Update incoming...